docs/pages/get_started/designers.js

```diff
@@ -85,7 +85,7 @@ Google Fonts
         <MainSection.Subsection
           title="Documentation"
           description={`
-Wondering how a component works? The dos and donts of its use?
+Wondering how a component works? The dos and dont's of its use?
```

Comment:
Should this be `do's and don't's`?

Reply:
or just `dos and don'ts`?

Reply:
according to the internets, `dos and don'ts`